src/Extensions/TimedScopes/ITimedScopeProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.TimedScopes
+{
+	/// <summary>
+	/// Interface to create TimedScope
+	/// </summary>
+	public interface ITimedScopeProvider
+	{
+		/// <summary>
+		/// Creates and start TimedScope
+		/// </summary>
+		TimedScope Start(TimedScopeDefinition name, TimedScopeResult result);
```

Comment:
nit: maybe a bit confusing that it's only called `Start` rather than `CreateAndStart`

Reply:
Makes sense to me, renamed, thanks